ang/civicase/ContactCard.html

```diff
@@ -54,7 +54,7 @@
           <i class="civicase__contact-icon-additional material-icons" ng-if="getContactIconOf(contact.contact_id) === 'Individual'">person</i>
           <i class="civicase__contact-icon-additional material-icons" ng-if="getContactIconOf(contact.contact_id) === 'Household'">home</i>
           <i class="civicase__contact-icon-additional material-icons" ng-if="getContactIconOf(contact.contact_id) === 'Organisation'">domain</i>
-          <span class="civicase__contact-name">{{ contact.display_name }}</span>
+          <span class="civicase__contact-name-additional">{{ contact.display_name }}</span>
```

Comment:
What is the use of `civicase__contact-name-additional` class here?

Reply:
It was necessary to add a separate class for contact names inside additional container, to stop bleeding styles applied to the contact name outside of additional container.

Reply:
Cool